src/main/java/uk/gov/pay/api/model/PaymentJSON.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.pay.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = "A Payment")
+interface PaymentJSON {
```

Comment:
Nice :+1:  that we have all the documentation annotations abstracted aways from implementation. (Although `Fran` tried this sometime ago and found out some complications in swagger. Not sure why though, but if this works then good)